src/TextCheckers.java

```diff
@@ -23,16 +17,26 @@ public static void main(String [] args) {
 	   System.out.println("" + c.getTurn() + "'s turn: ");
 	   do{
```

Comment:
You can avoid duplicating your condition if you amke the loop `while(true)` and use an `else {break;}` after your `if`.